go.mod

```diff
@@ -48,24 +46,12 @@ require (
 	github.com/mitchellh/reflectwalk v1.0.0 // indirect
 	github.com/pebbe/zmq4 v1.2.7 // indirect
 	github.com/pelletier/go-toml v1.9.4 // indirect
-	github.com/remyoudompheng/bigfft v0.0.0-20200410134404-eec4a21b6bb0 // indirect
 	github.com/x448/float16 v0.8.4 // indirect
 	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97 // indirect
 	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
 	golang.org/x/sys v0.0.0-20210806184541-e5e7981a1069 // indirect
 	golang.org/x/text v0.3.6 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	modernc.org/b v1.0.2 // indirect
-	modernc.org/db v1.0.3 // indirect
-	modernc.org/file v1.0.3 // indirect
-	modernc.org/fileutil v1.0.0 // indirect
-	modernc.org/golex v1.0.1 // indirect
-	modernc.org/internal v1.0.2 // indirect
-	modernc.org/lldb v1.0.2 // indirect
-	modernc.org/mathutil v1.4.1 // indirect
-	modernc.org/sortutil v1.1.0 // indirect
-	modernc.org/strutil v1.1.1 // indirect
-	modernc.org/zappy v1.0.3 // indirect
```

Comment:
please help remove all the libraries you deleted from https://github.com/edgexfoundry/device-virtual-go/blob/d36d4eb5d49c50cebb55fcc728bec397aa4f388e/Attribution.txt

Reply:
Removed - I checked against what was present after "go mod vendor".

Reply:
sorry, that how go 1.17 works, and I didn't notice that